src/commissaire/storage/kubernetes.py

```diff
@@ -0,0 +1,560 @@
+# Copyright (C) 2016  Red Hat, Inc
```

Comment:
I don't think we should bring in the kubernetes storage handler. At least not yet. I'd like us to just focus on the etcd backend and pull in others as the need arises.